examples/simple/src/main.rs

```diff
@@ -46,13 +44,18 @@ fn main() {
     vixen::Runtime::builder()
         .opts(config)
         .manager(HandlerManagers {
-            account: HandlerManager::new([
-                handler::boxed(vixen::HandlerPack::new(TokenExtensionProgramParser, [
-                    Handler,
-                ])),
-                handler::boxed(vixen::HandlerPack::new(TokenProgramParser, [Handler])),
-            ]),
-            transaction: HandlerManager::empty(),
+            account:
+            //  HandlerManager::new([
+            //     handler::boxed(vixen::HandlerPack::new(TokenExtensionProgramParser, [
+            //         Handler,
+            //     ])),
+            //     handler::boxed(vixen::HandlerPack::new(TokenProgramParser, [Handler])),
+            // ]),
+            HandlerManager::empty(),
+            transaction: HandlerManager::new([handler::boxed(vixen::HandlerPack::new(
+                TokenProgramIxsParser,
+                [Handler],
+            ))]),
```

Comment:
@ray-kast program's are only aware of instructions so we either need to make this instructions and then vixen needs to decompose the transaction and invoke instruction parsers and handlers. 

Other option is register all instruction parsers and then a single handler for the transaction. 

I think 1 is easiest for the integrator but using full transaction may support some use case by giving the integrator the whole transaction vs just pieces of it.